### PR TITLE
Add mongodb service

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

File: src/_base/_twig/docker-compose.yml/service/mongodb.yml.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    image: {{ @('services.mongodb.image') }}
+    environment: {{ to_nice_yaml(deep_merge([
+        @('services.mongodb.environment'),
+        @('services.mongodb.environment_secrets')
+      ]), 2, 6) | raw }}
+    labels:
+      - traefik.enable=false
+    networks:
+      - private
+    restart: always
```
Comment (kierenevans) [inline src/_base/_twig/docker-compose.yml/service/mongodb.yml.twig]: Is this intentional? Currently we only have that on workspace provided global services - traefik/proxy and mail

Comment (andytson-inviqa) [reply src/_base/_twig/docker-compose.yml/service/mongodb.yml.twig]: I can't remember

**Review by kierenevans — APPROVED** [review]

File: src/_base/helm/app/templates/service/mongodb/secret.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+{{ template "service.environment.secret" (dict "service_name" "mongodb" "service" .Values.docker.services.mongodb "Values" .Values) }}
```
Comment (kierenevans) [inline src/_base/helm/app/templates/service/mongodb/secret.yaml]:
```suggestion
{{ template "service.environment.secret" (dict "service_name" "mongodb" "service" .Values.docker.services.mongodb "root" $) }}
```

**Review by kierenevans — CHANGES_REQUESTED** [review]
The "root" suggestion is needed :+1:

**Review by kierenevans — APPROVED** [review]

**Pull request merged**